README.md

```diff
@@ -1,91 +1,209 @@
+For new users setting up Magda for the first time we recommend using these instructions - these use [Terraform](https://www.terraform.io/intro/index.html) to set you up with a instance running on Google Cloud Engine very quickly (about 5 minutes of entering commands / editing config and 20 minutes of waiting), and gives you a basic instance, and in another 30-60 minutes of waiting will get you HTTPS working on your own domain.
 
-2. Fork this repository - this means you can make your own customisations, but still pull in updates.
+If you want to run it locally in something like Minikube, or want to be a bit more direct, the old quickstart is still available at [legacy.md](./legacy.md).
 
-3. `git clone` it to your local machine open a terminal in the directory
+### 1. Clone this repo
 
-4. Look at config.yaml and make sure to customise the uncommented lines. Everything else can be left as default for now.
+```bash
+git clone https://github.com/magda-io/magda-config.git
+```
+
+or download it with the "Clone or download" button in Github.
+
+### 2. Install Terraform
+
+Go to [https://learn.hashicorp.com/terraform/getting-started/install.html](https://learn.hashicorp.com/terraform/getting-started/install.html) for instructions
+
+### 3. Install [Google Cloud SDK](https://cloud.google.com/sdk/docs/downloads-interactive)
+
+Go to [https://cloud.google.com/sdk/docs/downloads-interactive](https://cloud.google.com/sdk/docs/downloads-interactive) for instructions.
+
+Once `Google Cloud SDK` is installed, you also need to install gcloud beta components by the following command:
 
-5. Make sure your `kubectl` is connected to your kubernetes cluster and install helm
 ```bash
-kubectl create serviceaccount --namespace kube-system tiller
-kubectl create clusterrolebinding tiller-cluster-rule --clusterrole=cluster-admin --serviceaccount=kube-system:tiller
-helm init --service-account tiller
+gcloud components install beta
 ```
 
-6. Run the create secrets script in a command line and follow the prompts
+### 4. Create a Google Cloud Project
+
+Before you start the deployment process, you need to create a [google cloud project](https://cloud.google.com/resource-manager/docs/creating-managing-projects) via [Google Cloud Console](https://console.cloud.google.com/) and note down the `Project Id`. Note that this isn't necessarily exactly the same as the id you specified - if it's already been taken, Google will append some numbers to it. Make sure by checking the "Select a Project" dialog in Google Cloud:
+
+![Google Cloud Select a Project Dialog](./gcp-projects.png)
+
+### 5. Set Default Project
+
+Set the project id you noted down to an environment variable, because you'll need it in a few places - this will work in bash, if you're using another shell either figure out the equivalent or just manually replace $PROJECT_ID with your project id.
```

Comment:
```suggestion
Set the project id you noted down to an environment variable, because you'll need it in a few places - this will work in bash. If you're using another shell use the equivalent command or just manually replace `$PROJECT_ID` with your project id.
```